### PR TITLE
PHOENIX-2187: Adding front-end of Tracing Web App

### PR DESCRIPTION
**Pull request by AyolaJayamaha**
This will include the followingTracing visualization features.
- [x] List - lists the traces with their attributes
- [x] Trace Count - Chart view over the trace description
- [x] Dependency Tree - tree view of trace ids
- [x] Timeline - timeline of trace ids
- [x] Trace Distribution - Distribution chart of hosts of traces

### CODE REVIEW

**Comment by AyolaJayamaha** [comment]
Build phoenix-tracing-webapp by
`mvn package`
It will run jasmine JS Test Specs with testing JS files in Tracing Web app

screenshot from 2015-08-20 21 05 31

File: phoenix-tracing-webapp/pom.xml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    <description>Tracing web application will visualize the phoenix traces</description>
+
+    <properties>
+       <top.dir>${project.basedir}/..</top.dir>
+    </properties>
+
+    <dependencies>
+      <!--dependency>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-server</artifactId>
```
Comment (ndimiduk) [inline phoenix-tracing-webapp/pom.xml]: just remove these dependencies from this PR as they're provided in the other.

File: phoenix-tracing-webapp/src/build/build.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <project name="trace-webapp-demo" default="war" basedir=".">
+
+    <property name="project-name" value="${ant.project.name}" />
+    <property name="builder" value="ayola" />
+
+    <property name="war-file-name" value="${project-name}.war" />
+    <property name="webapp-directory" value="src/main/webapp" />
+    <property name="classes-directory" value="src/main/webapp" />
```
Comment (ndimiduk) [inline phoenix-tracing-webapp/src/build/build.xml]: this is provided in the other PR, right? no need to include it here.

**Comment by stoty** [comment]
Already merged.

**Pull request closed without merging**